docs/components/drawer/index.html

```diff
@@ -124,4 +124,78 @@ <h3 v-text="title"></h3>
         </footer>
       </div>
     </section>
+
+    <section>
+        <header>
+          <h2 id="basic-drawer">Positionable Descendants</h2>
```

Comment:
Please update `id` from `basic-drawer` to `drawer-positionable-elements`.

Reply:
Changes have been done, please review.